code/modules/mob/living/simple_animal/hostile/megafauna/captainarlem.dm

```diff
@@ -1,40 +1,84 @@
+	desc = "A grotesque, Enclave-manufactured and Enclave-loyal prime super mutant given cybernetic augmentations, an arm-mounted plasma repeater, and a hyper-advanced suit of hardened power armor sporting numerous upgrades and additions atop it's standard operational frame. Your ride ends here, mutie. Time to die."
 	health = 3000
 	maxHealth = 3000
 	icon_state = "arlem"
 	icon_living = "arlem"
 	icon_dead = "arlem"
 	icon = 'icons/fallout/mobs/bigmob.dmi'
-	friendly_verb_continuous = "mutters 'Enclave huh? Get out of here. This is a restricted area by order of President Eden.'"
-	friendly_verb_simple = "mutters 'Enclave huh? Get out of here. This is a restricted area by order of President Eden.'"
+	faction = list("enclave", "wastebot")
+	social_faction = list("enclave")
+	gender = "male"
+	friendly_verb_continuous = "mutters 'Up for lunch later?'"
+	friendly_verb_simple = "mutters 'Up for lunch later?'"
 	attack_verb_continuous = "eviscerates"
 	attack_verb_simple = "eviscerate"
 	attack_sound = 'sound/magic/demon_attack1.ogg'
 	weather_immunities = list("snow")
 	speak_emote = list("shouts")
-	armour_penetration = 0.7
+	armour_penetration = 0.5
 	melee_damage_lower = 40
-	melee_damage_upper = 50
+	melee_damage_upper = 40
 	vision_range = 30
 	aggro_vision_range = 30
-	speed = 1
+	speed = 2
 	move_to_delay = 8
 	ranged_cooldown_time = 10
-	rapid_melee = 16
+	minimum_distance = 1
+	rapid_melee = 2
 	melee_queue_distance = 20 // as far as possible really, need this because of charging
 	ranged = 1
-	pixel_x = -16
+	pixel_x = 0
 	wander = FALSE
 	movement_type = GROUND
 	var/charging = 0
 	blood_volume = BLOOD_VOLUME_NORMAL
-	deathmessage = "<span class='colossus'>chokes on their own blood, gurgling out 'You're coming with me, mutie' as they activate their self destruct!</span>"
+	deathmessage = "chokes on his own blood, falling over as he activates his suit's self destruct!"
 	deathsound = 'sound/effects/gravhit.ogg'
 	del_on_death = FALSE
-	loot = list(/obj/item/keycard/library)
-
+	loot = list(/obj/item/keycard/library, /obj/effect/gibspawner/human, /obj/effect/temp_visual/gib_animation)
+	attack_action_types = list(/datum/action/innate/megafauna_attack/charge,
+							   /datum/action/innate/megafauna_attack/triple_charge,
+							   /datum/action/innate/megafauna_attack/blast,
+							   /datum/action/innate/megafauna_attack/eyebots)
+	var/list/spans_list = list("danger", "big")
+/* Cooldowns for abilities */
+	var/charge_cooldown = 0
+	var/blast_cooldown = 0
+	var/eyebots_cooldown = 0
+/*  Actual cooldown times  */
+	var/charge_cooldown_time = 4 SECONDS
+	var/blast_cooldown_time = 15 SECONDS
+	var/eyebots_cooldown_time = 30 SECONDS
+
+/datum/action/innate/megafauna_attack/charge
+	name = "Charge"
+	icon_icon = 'icons/mob/actions/actions_spells.dmi'
+	button_icon_state = "icebeam0"
+	chosen_message = "<span class='colossus'>You will now dash to your target.</span>"
+	chosen_attack_num = 1
```

Comment:
Could you use defines instead of magic numbers?

Reply:
Defines where exactly and for what.

Reply:
Look at `HERALD_TRISHOT`, `TENTACLE_PATCH` and `LEGIONNAIRE_CHARGE` for examples.